linguatec_lexicon/views.py

```diff
@@ -105,11 +105,12 @@ def near(self, request):
     @action(detail=False)
     def search(self, request):
         query = self.request.query_params.get('q', None)
+        lex = self.request.query_params.get('l', None)
         if query is not None:
             query = query.strip()
-
-        queryset = Word.objects.search(query)
-
+        if lex is not None:
+            lex = lex.strip()
+        queryset = Word.objects.search(query, lex)
```

Comment:
If you replace line 108 with:
```python
lex = self.request.query_params.get('l', '')    # default empty string intead of None
```
You could remove line #111 (conditional), because `strip()` could be always run without errors.

Reply:
Yeah, it is probably better to set the default value to '', change made and tested, it works fine.